tracelogging/example/src/tracelogging_example.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Windows.h"
+#include <g3log/g3log.hpp>
+#include <g3log/logworker.hpp>
+#include <g3log_tracelogging/g3log_tracelogging.h>
+#include <TraceLoggingProvider.h>
```

Comment:
Is `TraceLoggingProvider.h` available on Windows by default?
If so, would you be interested in working with me to get this code working under a cloud CI/Windows?
I.e similar to how g3log PRs work now: **https://github.com/KjellKod/g3log/blob/master/appveyor.yml**
(it should be done in a different PR for sure ... but would be absolutely kick ass if we could get this under CI build coverage )

Reply:
Yes, I'm pretty sure it is available on Windows by default.  Yes, I would definitely be interested in working with you to get it working in CI.  I'm completely new to appveyor but I'll take a look and see what I can figure out.